src/types/types.ts

```diff
@@ -135,6 +135,8 @@ export interface PlayerProgressbarOptions {
  * @property {number} [bufferingTimeout=3000] Buffering timeout for the stream
  * @property {boolean} [spotifyBridge=true] If player should bridge spotify source to youtube
  * @property {boolean} [disableVolume=false] If player should disable inline volume
+ * @property {boolean} [volumeSmoothness=0] The volume transition smoothness between volume changes (lower the value to get better result)
```

Comment:
this is by default 0 on `Queue`, whereas `Player.createQueue` sets it to `0.1` by default